lib/ui/src/components/sidebar/treeview/treeview.js

```diff
@@ -334,7 +351,7 @@ class TreeState extends PureComponent {
         {Filter ? <Filter key="filter" onChange={this.events.onFilter} /> : null}
         {roots.length || others.length ? (
           <Fragment>
-            {roots.map(({ id, name, children }) => (
+            {roots.sort(storySort).map(({ id, name, children }) => (
```

Comment:
looks like that the sort will happen on every render, IMO not the best thing to do

Reply:
Agreed, had proposed it over in the issue with the same concern. Was missing the info that @tmeasday just offered up though, so now I like it even less :)
I'll try to look at the file he offered up soon.